[2022-08-29 00:00:02,635] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.model_dag scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29 00:00:02,644] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.model_dag scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29 00:00:02,644] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 00:00:02,645] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-29 00:00:02,646] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 00:00:02,659] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): model_dag> on 2022-08-28 00:00:00+00:00
[2022-08-29 00:00:02,664] {standard_task_runner.py:52} INFO - Started process 10682 to run task
[2022-08-29 00:00:02,667] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'model_dag', 'scheduled__2022-08-28T00:00:00+00:00', '--job-id', '275', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmp2r22wwig', '--error-file', '/tmp/tmps_bftalc']
[2022-08-29 00:00:02,668] {standard_task_runner.py:80} INFO - Job 275: Subtask model_dag
[2022-08-29 00:00:02,719] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.model_dag scheduled__2022-08-28T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-29 00:00:02,781] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=model_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-28T00:00:00+00:00
[2022-08-29 00:00:44,769] {python.py:173} INFO - Done. Returned value was: None
[2022-08-29 00:00:44,780] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=model_dag, execution_date=20220828T000000, start_date=20220829T000002, end_date=20220829T000044
[2022-08-29 00:00:44,814] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29 00:00:44,838] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
